README.md

```diff
@@ -50,7 +50,8 @@ Please send us a pull request with your suggestions.
 
 #### Doesn't this impact the right to free speech? Why are you stomping on my rights?
 
-See this XKCD for our feelings on this matter: https://xkcd.com/1357/
+In the many countries that do have the right to free speech, this means (with obvious exeptions) the government cannot arrest you for saying anything.  Many countries contain free speech in their constitution which can only apply to the government for enforcing or creating laws.
+As most companies and organizations are owned by private individuals, they are only restricted by laws that apply to people and their businesses.  The free speech sections contained in constitutions only apply to the government, restricting the kinds of laws that it can make.  So no, this does not impact your right to free speech.
```

Comment:
@Addvilz Free speech is important, I'm with you on that, but none of this is "public". This is github we're talking about. If you're discussing race, gender, etc you're doing it wrong. I'm not aware of any library/framework/language that requires you to discuss your own or inquire or critique anyone else's. I mean positively or negatively it's rarely a necessity to develop great software.

This only means you can't mock or harass others based on things completely unrelated to their work. You're entitled to your opinion, but the CoC is happening for better or worse. @KelseyDH raises a great point re: opal. You want your free speech, I support that, but the CoC should protect that. You can dissent against your government, disagree publicly with the popular opinion, and be confident that, so long as you don't let it bleed into your pull requests, you can't be removed as a maintainer because someone disagrees with you. 

So look hard at the CoC, does it protect you from repercussions for opinions you have, but keep separate from the issue tracker? If it doesn't you should propose a change, at least point it out. The CoC should protect all of us, not just some of us. 

Just like the issue on the opal tracker, you don't want someone swooping in and harassing you because they found out you held an (in the opal case an IMO admittedly distasteful) opinion unrelated to your work.